packages/sync-core/src/lib/diff.ts

```diff
@@ -182,7 +182,7 @@ export interface ObjectDiff {
  * @internal
  */
 export function diffRecord(prev: object, next: object): ObjectDiff | null {
-	return diffObject(prev, next, new Set(['props']))
+	return diffObject(prev, next, new Set(['props', 'meta']))
```

Comment:
@ds300 fyi, would like to include this. Do not intend to sneak this in.

Reply:
good one